.docker/Dockerfile-cms

```diff
@@ -266,7 +267,8 @@ RUN chmod -R +x /etc/periodic \
     && echo "    containertag: " $CONTAINERTAG >> /etc/motd \
     && echo >> /etc/motd \
     && cp /etc/motd /var/www/web/version.txt \
-    && echo "PATH=$PATH" > /etc/profile.d/drupal.sh
+    && echo "PATH=$PATH" > /etc/profile.d/drupal.sh \
+    && echo "alias tll='less $(ls -tr *.log | tail -1)'" > /etc/profile.d/usagov-aliases.sh
```

Comment:
Testing this on a local build, I see two problems: 

1. The alias isn't written out properly; I get `alias tll='less '` We need a backslash before the $. 
2. Once I've added the backslash, `tll` still doesn't do what's desired. I think what we really want is `tail -f $(ls -tr *.log | tail -1)`

Reply:
fixed and tested on Mac